Chapters/2-2-Chapter9.md

```diff
@@ -0,0 +1,169 @@
+
+"Kipnu! Get up!" A voice called to him from behind the fabric. It was Setre.
+
+She burst into the room, in full armor and spear, and didn't hesitate to pull Kipnu up to his feet. The blood suddenly rushed out of his head and he swayed to and fro to gain his balance amidst the piercing pain in his head. His eyes were open yet he still saw nothing, not even the blue light from Setre's chest.
+
+"It's time to go," she said in a hurried tone.
+
+He pushed her away with what little strength he had.
+
+"What happened to me?" he shouted at her, holding his head in agony.
+
+"The kanosin will wear off" she said, some trouble in her voice.
+
+"I need more... make it stop."
+
+"Nevermind that, you have to come out right now."
+
+"No," he responded harshly.
+
+She pulled her spear on him, and he swatted it away.
+
+"How do you threaten me at a time like this?" He asked rhetorically.
+
+With a flick of the wrist, she made a shallow slice through a fatty part of his hip, making Kipnu shout in pain.
+
+"With a spear," she said.
+
+She sighed and pulled out a cloth wrapping from her sling bag and threw it at him.
+
+"Bandage yourself up and come back into the Great Pavilion. Shinna will speak to you."
+
+"I have nothing to talk about."
+
+"You won't be talking."
+
+She left the room, leaving a still-bleeding Kipnu barely standing. He bandaged the wound the way he had done for many of his patients, but it was markedly more difficult to do it for himself in such a state. Once bandaged, he realized that he was still freezing and without clothing. _This must be some initiation or something. I can't just leave, I have nowhere to go._ He stood himself up and walked to the entryway, reaffirming his own existence before he went into the next room.
+
+The room had nearly hundreds of people forming two lines that surrounded Shinna and him. They all had the same blue luminescent markings as yesterday but they were silent now, staring at him like he was some prize. _Why is everyone here? What are they looking at me for?_
+
+Shinna and hae crew sat where he entered the pavilion yesterday, the rest forming a two lines, a tunnel of people surrounding him on both sides.
+
+"Kipnu. Approach," Shinna commanded, hae voice carrying well from across the room. Setre was behind him, ensuring he didn't turn back.
+
+He turned his head, but kept walking, "Setre, why all this ceremony for me?"
+
+"You are healthy, strong, and you are a doctor from Heiko. People like you don't come here often."
+
+"What makes a doctor so valuable?"
+
+"There are things we don't yet understand about this place. How it... affects us."
+
+As he approached, the people followed behind him to form an oval. Once he had gotten close enough Setre instructed him to kneel, which he did slowly.
+
+Shinna began the ceremony, "Tribes of Ulu. Yesterday, my ever-faithful hunter, Setre, found and claimed the man you see before you."
```

Comment:
There's multiple tribes? Are the people who are here one tribe or a union of tribes?

Reply:
multiple tribe thing is too confusing. Sticking to one big one.